<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Backoffice</class>
 <widget class="QMainWindow" name="Backoffice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backoffice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget_produits">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_rayons">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>421</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_produit_image">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>Produit_image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>28</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_accepted_produit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Accepted</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_denied_produit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Denied</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_produit_nom">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_produit_description">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_productName">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Product name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_productDesc">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Product description :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_produit_pu">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PU : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_produit_stock">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock : </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
